cmd/boot-script-service/main.go

```diff
@@ -417,9 +419,11 @@ func parseCmdLine() {
 	flag.StringVar(&bssdbName, "postgres-dbname", bssdbName, "(BSS_DBNAME) Postgres database name")
 	flag.StringVar(&sqlUser, "postgres-username", sqlUser, "(BSS_DBUSER) Postgres username")
 	flag.StringVar(&sqlPass, "postgres-password", sqlPass, "(BSS_DBPASS) Postgres password")
+	flag.StringVar(&jwksURL, "jwks-url", jwksURL, "Set the JWKS URL to fetch the public key for authorization")
 	flag.BoolVar(&insecure, "insecure", insecure, "(BSS_INSECURE) Don't enforce https certificate security")
 	flag.BoolVar(&debugFlag, "debug", debugFlag, "(BSS_DEBUG) Enable debug output")
 	flag.BoolVar(&useSQL, "postgres", useSQL, "(BSS_USESQL) Use Postgres instead of ETCD")
+	flag.BoolVar(&requireAuth, "require-auth", requireAuth, "Require JWTs authorization to allow using API endpoint")
```

Comment:
Similar to the comment regarding `retryAttempts`, we should also create an environment variable for this flag.